hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.test.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react";
+import CartSummary from "components/cart/CartSummary/CartSummary";
+import { cartQuantity, mockCartItems } from "testing/mockData";
+import store, { makeStore, RootState } from "slices/store";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+import thunk, { ThunkDispatch } from "redux-thunk";
+import {
+    initialState,
+    cartReducerName,
+    submitOrder,
+    cartSliceSelector,
+    cartSelectors,
+} from "slices/hardware/cartSlice";
+import { post } from "api/api";
+import { displaySnackbar } from "slices/ui/uiSlice";
+import { makeMockApiListResponse } from "testing/utils";
+import { AnyAction } from "redux";
+
+it(`Renders correctly when it reads the number ${cartQuantity}`, async () => {
```

Comment:
why is this test floating up here lol. Could you place it in a describe block below?

Reply:
A test you should add here is when clicking "Submit Order" should make the correct API post request and the cart store should be cleared. You can look at Inventory.test.tsx for inspiration because you'll need to mock the api calls.

Reply:
> why is this test floating up here lol. Could you place it in a describe block below?

good question lol

Reply:
> A test you should add here is when clicking "Submit Order" should make the correct API post request and the cart store should be cleared. You can look at Inventory.test.tsx for inspiration because you'll need to mock the api calls.

Where can I look up the response for the POST request, so I can mock it?

Reply:
It's the post request you wrote the thunk you, for submitting the order